desktop/mac/index.md

```diff
@@ -227,9 +225,20 @@ For information about the reset options, see [Logs and Troubleshooting](troubles
 
 ### Software Updates
 
-The Software Updates section notifies you of any updates available to Docker Desktop. You can choose to download the update right away, or click the **Release Notes** option to learn what's included in the updated version.
+The **Software Updates** section notifies you of any updates available to Docker Desktop. When there's a new update, you can choose to download the update right away, or click the **Release Notes** option to learn what's included in the updated version.
+
+> **Disable automatic check for updates**
+>
+> Starting with Docker Desktop 4.2.0, the option to turn off the automatic check for updates is available for users on all Docker subscriptions, including Docker Personal. Update to [Docker Desktop 4.2.0](../mac/release-notes/index.md) to start using this feature.
+{: .important}
+
+Turn off the check for updates by clearing the **Automatically Check for Updates** check box. This disables notifications in the Docker menu and also the notification badge that appears on the Docker Dashboard. To check for updates manually, select the **Check for updates** option in the Docker menu.
+
+To allow Docker Desktop to automatically download new updates in the background, select **Always download updates**. This downloads newer versions of Docker Desktop when an update becomes available.
 
-If you are on a Docker Team or a Business subscription, you can turn off the check for updates by clearing the **Automatically Check for Updates** checkbox in the [General](#general) settings. This will also disable the notification badge that appears on the Docker Dashboard.
+Select **Use nightly builds** to download nightly builds instead of stable builds When you check for available updates.
```

Comment:
@usha-mandya I think this is still behind a feature flag so users will have to enable it. @mat007 Am I right about that?

Reply:
Yes, it’s behind a flag. I don’t think we should mention that option in the docs until/whenever (if) we decide to enable it for everyone.

Reply:
Aah OK. I'll remove this

Reply:
Fixed this.